.github/workflows/tests.yml

```diff
@@ -73,6 +73,8 @@ jobs:
     - name: Add config
       run: |
         echo "[gallia]" > gallia.toml
+        echo "verbosity = 10" >> gallia.toml
+        echo "no-volatile-info = true" >> gallia.toml
```

Comment:
The volatile logging needs auto terminal detection; this should not be needed IMO.

Reply:
Not sure what you mean?
I don't know how github jobs are set up and what should be done here...

In any case I see this as not related to this MR. Feel free to vote for dropping edef9290411819fb7cc4d2bea237db58b73c9909 again, I was just trying to debug the failing CI job...